### ISSUE TITLE
Can't install with Spree 3.4.1

### ISSUE THREAD

**Issue by danielmyasnikov** [spree-contrib/spree_globalize#41]
Here is the error:

```
Bundler could not find compatible versions for gem "spree":
  In snapshot (Gemfile.lock):
    spree (= 3.4.1)

  In Gemfile:
    spree (~> 3.4.0)

    spree_globalize was resolved to 3.3.0, which depends on
      spree_i18n was resolved to 1.0.0, which depends on
        spree (~> 1.1)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by willwoodlief** [comment]
I tried installing on a fresh generated rails 5.1.4 app with the following gem file

```
source 'https://rubygems.org'

git_source(:github) do |repo_name|
  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
  "https://github.com/#{repo_name}.git"
end


# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
gem 'rails', '~> 5.1.4'
# Use mysql as the database for Active Record
gem 'mysql2', '>= 0.3.18', '< 0.5'
# Use Puma as the app server
gem 'puma', '~> 3.7'
# Use SCSS for stylesheets
gem 'sass-rails', '~> 5.0'
# Use Uglifier as compressor for JavaScript assets
gem 'uglifier', '>= 1.3.0'
# See https://github.com/rails/execjs#readme for more supported runtimes
# gem 'therubyracer', platforms: :ruby

# Use CoffeeScript for .coffee assets and views
gem 'coffee-rails', '~> 4.2'
# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
gem 'turbolinks', '~> 5'
# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
gem 'jbuilder', '~> 2.5'
# Use Redis adapter to run Action Cable in production
# gem 'redis', '~> 3.0'
# Use ActiveModel has_secure_password
# gem 'bcrypt', '~> 3.1.7'

# Use Capistrano for deployment
# gem 'capistrano-rails', group: :development
#

gem "awesome_print", require:"ap"

group :development, :test do
  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
  # Adds support for Capybara system testing and selenium driver
  gem 'capybara', '~> 2.13'
  gem 'selenium-webdriver'
  gem 'railroady' # for making diagrams of models and controllers https://github.com/preston/railroady
end

group :development do
  # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
  gem 'web-console', '>= 3.3.0'
  gem 'listen', '>= 3.0.5', '< 3.2'
  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
  gem 'spring'
  gem 'spring-watcher-listen', '~> 2.0.0'
end

# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]


#spree

gem 'spree', '~> 3.3'
gem 'spree_auth_devise', '~> 3.3'
gem 'spree_gateway', '~> 3.3'
gem 'spree_i18n', github: 'spree-contrib/spree_i18n'
gem 'spree_globalize', github: 'spree-contrib/spree_globalize', branch: 'master'
```

And I got the following error, after emptying out my gemset and doing a bundle install

```
Bundler could not find compatible versions for gem "activerecord":
  In Gemfile:
    spree_globalize was resolved to 3.3.1, which depends on
      globalize was resolved to 3.0.3, which depends on
        paper_trail (~> 2) was resolved to 2.7.2, which depends on
          activerecord (~> 3.0)

    rails (~> 5.1.4) was resolved to 5.1.4, which depends on
      activerecord (= 5.1.4)

Bundler could not find compatible versions for gem "globalize":
  In Gemfile:
    spree_globalize was resolved to 3.3.1, which depends on
      globalize

Could not find gem 'globalize', which is required by gem 'spree_globalize', in any of the sources.

Bundler could not find compatible versions for gem "rails":
  In Gemfile:
    spree_globalize was resolved to 3.3.1, which depends on
      globalize was resolved to 3.0.3, which depends on
        paper_trail (~> 2) was resolved to 2.6.2, which depends on
          rails (~> 3)

    spree (~> 3.3) was resolved to 3.3.0, which depends on
      spree_core (= 3.3.0) was resolved to 3.3.0, which depends on
        rails (~> 5.1.1)

Bundler could not find compatible versions for gem "spree_core":
  In Gemfile:
    spree (~> 3.3) was resolved to 3.3.0, which depends on
      spree_core (= 3.3.0)

    spree_globalize was resolved to 3.3.1, which depends on
      spree_i18n was resolved to 3.3.2, which depends on
        spree_core (< 4.0, >= 3.1.0)
```

**Comment by einSelbst** [comment]
Please see this comment from another issue here, it fixes the problem for me:
https://github.com/spree-contrib/spree_globalize/issues/34#issuecomment-324972359

**Comment by restot** [comment]
its not help, on spree 3.4.4

**Comment by ramesh2007** [comment]
Same issue here but they said @fixgit issue solve 

rails - 5.1.4
spree -3.3

Bundler could not find compatible versions for gem "spree":
In Gemfile:
spree (~> 3.3)

Could not find gem 'spree (~> 3.3)' in any of the sources.

Bundler could not find compatible versions for gem "spree_core":
In Gemfile:
spree (~> 3.3) was resolved to 3.3.0, which depends on
spree_core (= 3.3.0)

spree_redirects was resolved to 0.3.0, which depends on
  spree_core (~> 1.1)

**Comment by exocode** [comment]
Of course also not working with 3.6.4. (I also added `gem 'activemodel-serializers-xml'` like suggested above, which didn't help).

```
/Users/janjezek/.rbenv/versions/2.5.1/bin/ruby /Users/janjezek/Coding/RubymineProjects/demo52-spree/bin/bundle update spree_core
Fetching https://github.com/spree-contrib/spree_globalize.git
The dependency tzinfo-data (>= 0) will be unused by any of the platforms Bundler is installing for. Bundler is installing for ruby but the dependency is only for x86-mingw32, x86-mswin32, x64-mingw32, java. To add those platforms to the bundle, run `bundle lock --add-platform x86-mingw32 x86-mswin32 x64-mingw32 java`.
Fetching gem metadata from https://rubygems.org/...........
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies.....
Bundler could not find compatible versions for gem "activerecord":
  In Gemfile:
    spree (~> 3.6.4) was resolved to 3.6.4, which depends on
      spree_core (= 3.6.4) was resolved to 3.6.4, which depends on
        acts-as-taggable-on (~> 6.0) was resolved to 6.0.0, which depends on
          activerecord (~> 5.0)

    spree (~> 3.6.4) was resolved to 3.6.4, which depends on
      spree_core (= 3.6.4) was resolved to 3.6.4, which depends on
        acts_as_list (~> 0.8) was resolved to 0.9.16, which depends on
          activerecord (>= 3.0)

    spree (~> 3.6.4) was resolved to 3.6.4, which depends on
      spree_core (= 3.6.4) was resolved to 3.6.4, which depends on
        awesome_nested_set (~> 3.1.3) was resolved to 3.1.4, which depends on
          activerecord (< 5.3, >= 4.0.0)

    spree (~> 3.6.4) was resolved to 3.6.4, which depends on
      spree_core (= 3.6.4) was resolved to 3.6.4, which depends on
        friendly_id (~> 5.2.1) was resolved to 5.2.4, which depends on
          activerecord (>= 4.0.0)

    spree_globalize was resolved to 3.3.1, which depends on
      globalize was resolved to 4.0.0.alpha.5, which depends on
        paper_trail (~> 3.0.0.beta) was resolved to 3.0.9, which depends on
          activerecord (< 5.0, >= 3.0)

    spree (~> 3.6.4) was resolved to 3.6.4, which depends on
      spree_core (= 3.6.4) was resolved to 3.6.4, which depends on
        paranoia (~> 2.4.1) was resolved to 2.4.1, which depends on
          activerecord (< 5.3, >= 4.0)

    rails (~> 5.2.1) was resolved to 5.2.1, which depends on
      activerecord (= 5.2.1)

    spree (~> 3.6.4) was resolved to 3.6.4, which depends on
      spree_core (= 3.6.4) was resolved to 3.6.4, which depends on
        ransack (~> 2.0.1) was resolved to 2.0.1, which depends on
          activerecord (>= 5.0)

    spree_gateway (~> 3.3) was resolved to 3.3.3, which depends on
      spree_extension was resolved to 0.0.5, which depends on
        activerecord (>= 4.2)

    spree (~> 3.6.4) was resolved to 3.6.4, which depends on
      spree_core (= 3.6.4) was resolved to 3.6.4, which depends on
        state_machines-activerecord (~> 0.5) was resolved to 0.5.1, which depends on
          activerecord (< 6.0, >= 4.1)

Bundler could not find compatible versions for gem "rails":
  In Gemfile:
    spree (~> 3.6.4) was resolved to 3.6.4, which depends on
      spree_frontend (= 3.6.4) was resolved to 3.6.4, which depends on
        canonical-rails (~> 0.2.3) was resolved to 0.2.4, which depends on
          rails (< 5.3, >= 4.1)

    spree_globalize was resolved to 3.3.1, which depends on
      globalize was resolved to 3.0.3, which depends on
        paper_trail (~> 2) was resolved to 2.6.2, which depends on
          rails (~> 3)

Process finished with exit code 6

```

**Comment by damianlegawiec** [comment]
Finally fixed :)
